companion-api/README.md

```diff
@@ -11,9 +11,9 @@ companion-api
 #### With etcd backend
 
 ```sh
-AUTH_BACKEND=etcd \
-ETCD_ENDPOINTS=http://localhost:2379 \
-ETCD_PREFIX=/users \
+export AUTH_BACKEND=etcd \
```

Comment:
no it was ok without export (this is the single-line exec syntax to define env for something you run)